bin/node/src/commands/store.rs

```diff
@@ -119,7 +128,29 @@ impl StoreCommand {
             .as_secs()
             .try_into()
             .expect("timestamp should fit into u32");
-        let genesis_state = GenesisState::new(vec![account_file.account], version, timestamp);
+        Ok(GenesisState::new(vec![account_file.account], version, timestamp))
+    }
+
+    fn bootstrap(
+        data_directory: &Path,
+        accounts_directory: &Path,
+        genesis_config: Option<&PathBuf>,
+    ) -> anyhow::Result<()> {
+        let genesis_state = if let Some(genesis_config) = genesis_config {
+            let toml_str = fs_err::read_to_string(genesis_config)?;
+            let config = GenesisConfig::read_toml(toml_str.as_str())
+                .context(format!("Read from file: {}", genesis_config.display()))?;
+
+            let (state, secrets) = config.into_state()?;
+            // Write the accounts to disk
+            for AccountFileWithName { account_file, name } in secrets.as_account_files() {
+                let accountpath = accounts_directory.join(name);
+                account_file.write(accountpath)?;
+            }
+            state
+        } else {
+            Self::default_genesis_state(accounts_directory)?
+        };
```

Comment:
WDYT about something more like this? 

```suggestion
        let config = genesis_config
            .map(GenesisConfig::from_toml_file)
            .transpose()?
            .unwrap_or_default();
            
            let (state, secrets) = config.into_state()?;
            // Write the accounts to disk
            for AccountFileWithName { account_file, name } in secrets.as_account_files() {
                let accountpath = accounts_directory.join(name);
                account_file.write(accountpath)?;
            }
            state
```

Reply:
Implemented